README.md

```diff
@@ -267,7 +270,7 @@ A few notes:
   you will be able to run only a single FPC development container on a
   particular host.
 
-To build the docker image, run
+Once you have cloned the repository, to build the docker image navigate to the utils/docker subdirectory and run
```

Comment:
```suggestion
Once you have cloned the repository, to build and run the FPC docker container run
```

Reply:
I suggest that we merge the next two code blocks and just keep 
```bash
cd utils/docker; make run
```

Reply:
Done!